src/Analyzers/CSharp/Tests/RemoveUnnecessarySuppressions/RemoveLegacySuppressionsTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.RemoveUnnecessarySuppressions;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+using VerifyCS = Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions.CSharpCodeFixVerifier<
+    Microsoft.CodeAnalysis.CSharp.RemoveUnnecessarySuppressions.CSharpRemoveUnnecessarySuppressionsDiagnosticAnalyzer,
+    Microsoft.CodeAnalysis.RemoveUnnecessarySuppressions.RemoveLegacySuppressionsCodeFixProvider>;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.RemoveUnnecessarySuppressions
+{
+    [Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnnecessarySuppressions)]
+    [WorkItem(44362, "https://github.com/dotnet/roslyn/issues/44362")]
+    public class RemoveLegacySuppressionsTests
+    {
+        [Fact]
+        public void TestStandardProperties()
+            => VerifyCS.VerifyStandardProperties();
+
+        // Namespace
+        [InlineData("namespace", "N", "~N:N")]
```

Comment:
what does `~` mean btw?

Reply:
This is an optional prefix to Roslyn doc comment ID based suppressions, I think it was added to differentiate between Roslyn and legacy suppressions, but I do not know for sure. This is the format that IDE suppression code fixes use, so I have retained it for this migration code fix.